README.md

```diff
@@ -11,8 +11,8 @@ rev-ruby-sdk
 
 ### Background
 
-Rev.com provides transcription and translation services powered by humans. The Rev.com API allows you to tap these services
-with no manual steps. It's a RESTful API, documented at <https://www.rev.com/api/docs>.
+Rev.com provides caption, transcription and translation services powered by humans. The Rev.com API allows you to tap caption
```

Comment:
When communicating externally, we always use TC, CP, TR ordering.  So here it is transcription, caption, and translation services..... tap transcription and caption services ....